pyscipopt/expr.pxi

```diff
@@ -135,7 +132,12 @@ cdef class Expr:
         '''turn it into a constraint'''
         return _expr_richcmp(self, other, op)
 
+    def normalize(self):
```

Comment:
Why did you move the method down?
I think you can't rename it, because self.expr._normalize() is still called further down. Hopefully, some test should fail now.